.deployment.config.json

```diff
@@ -71,6 +71,17 @@
       },
       {
         "system_certifier": "snyk-code"
+      },
+      {
+        "github": {
+          "name": "e2e-certifier",
+          "workflow_repository": "coveo/search-ui",
```

Comment:
I'm not certain that the deployment pipeline will accept running a github certifier on a public repo. If not, we can move the job to a private repo and specify it here. This would also let us use a Coveo Github action runner, which we can't in this repo.

Reply:
Can work.
Issue mid-term will be triggering the pipeline (i.e. removing Jenkins)

Reply:
If the Jenkins version of the deployment-launching script is https://github.com/coveo/search-ui/blob/master/Jenkinsfile, it looks like it shouldn't be too hard to convert to a Github action. It's pretty much all simple bash commands and calls to other scripts in this repo. (But like you said that's outside the scope of this PR.)